jwst/outlier_detection/outlier_detection.py

```diff
@@ -497,7 +497,7 @@ def flag_cr(sci_image, blot_image, **pars):
     log.debug("Pixels in cr_mask:  {}".format(count_cr))
 
     # Update the DQ array in the input image in place
-    np.bitwise_or(sci_image.dq, np.invert(cr_mask) * CRBIT, sci_image.dq)
+    sci_image.dq = np.bitwise_or(sci_image.dq, np.invert(cr_mask) * CRBIT)
```

Comment:
Hey does this fix a known bug?!

Reply:
Actually no and this was one of the issues to be brought up during the team meeting but time ran out. Before this PR, all data arrays are loaded by this point, so numpy functions that allow operation in-place work fine. However, with this PR, having the data arrays loaded are not guaranteed. Thus, at this point the numpy arrays can get a `Block` node object. This routine does a check to ensure that the output can be written to, which now fails because of the unknown `Block` object type.